von-x-agent/config/services.yml

```diff
@@ -33,7 +33,7 @@ issuers:
     credential_types:
     - description: Permit
       schema: my-permit.my-organization
-      issuer_url: $ENDPOINT_URL/my-organization/my-permit
+      issuer_url: $APPLICATION_URL/my-organization/my-permit
```

Comment:
I don't think this works.  If APPLICATION_URL is not passed in, it will be http://localhost:5000, but the von-agent-template agent is at http://localhost:5001.  I think that's why it was using ENDPOINT_URL.

5000 is for all the agents pre-defined in dFlow.

I don't know where the 10.10.10.246 value came from.

@esune @ianco

Reply:
I tested it and it works locally. The reason is `WEB_HTTP_PORT` is set to `5001` a couple of instructions before `APPLICATION_URL` is set in the `manage` script. I pushed a small update to make things consistent as this is confusing though.

Reply:
Tested locally, everything is working